docs/feature_rgblight.md

```diff
@@ -79,28 +79,40 @@ Your RGB lighting can be configured by placing these `#define`s in your `config.
 
 ## Animations
 
-Not only can this lighting be whatever color you want, if `RGBLIGHT_ANIMATIONS` is defined, you also have a number of animation modes at your disposal:
-
-|Mode |Description          |
-|-----|---------------------|
-|1    |Solid color          |
-|2-5  |Solid color breathing|
-|6-8  |Cycling rainbow      |
-|9-14 |Swirling rainbow     |
-|15-20|Snake                |
-|21-23|Knight               |
-|24   |Christmas            |
-|25-34|Static gradient      |
-|35   |RGB Test             |
-|36   |Alternating          |
+
+Not only can this lighting be whatever color you want,
+if `RGBLIGHT_EFFECT_xxxx` or `RGBLIGHT_ANIMATIONS` is defined, you also have a number of animation modes at your disposal:
+
+|Mode                                   |Description          |
+|---------------------------------------|---------------------|
+|`RGBLIGHT_MODE_STATIC_LIGHT`           |Solid color          |
+|`RGBLIGHT_MODE_BREATHING + 0-3`        |Solid color breathing|
+|`RGBLIGHT_MODE_RAINBOW_MOOD + 0-2`     |Cycling rainbow      |
+|`RGBLIGHT_MODE_RAINBOW_SWIRL + 0-5`    |Swirling rainbow     |
+|`RGBLIGHT_MODE_SNAKE + 0-5`            |Snake                |
+|`RGBLIGHT_MODE_KNIGHT + 0-2`           |Knight               |
+|`RGBLIGHT_MODE_CHRISTMAS`              |Christmas            |
+|`RGBLIGHT_MODE_STATIC_GRADIENT + 0-9`  |Static gradient      |
+|`RGBLIGHT_MODE_RGB_TEST`               |RGB Test             |
+|`RGBLIGHT_MODE_ALTERNATING`            |Alternating          |
+
 
 Check out [this video](https://youtube.com/watch?v=VKrpPAHlisY) for a demonstration.
 
 The following options can be used to tweak the various animations:
```

Comment:
It may be worth at least noting in the documentation that while most of the modes can now be disabled at compile time, it is not possible to disable the static solid color mode.

Reply:
I added '(this mode always enable)' to static color mode.
How about this?